Sources/Identity/IdentityManager.swift

```diff
@@ -100,10 +129,10 @@ class IdentityManager: CurrentUserProvider {
         }
     }
 
-    func logOut(completion: (Error?) -> Void) {
+    func performLogOut(completion: (Error?) -> Void) {
```

Comment:
This could use a better name too.

Reply:
There are 2 `logOut` methods now. The `internal` one performs the sync first.

Reply:
I'm not a big fan of that cause that's exposing an implementation detail in the name :/

Reply:
Yeah, I'm on the fence, though. If it's internal, the naming actually serves to remind us why we have it. If it were public API I'd definitely say you're 100% correct.

Reply:
Yeah, ultimately, I don't have a strong preference for non-public api naming here. What you have `perform` is totally workable IMO